### DISCUSSION TITLE
Different Languages, interface translations

### DISCUSSION THREAD

**Discussion by bouillon** [itswadesh/svelte-commerce#102] [General]
Hi, 

I'm looking for a shop-front end with modern Framework.

This looks quite interesting. I'm asking me it there some translations for different languages like German, French.
This seems not to be the case.

I see an open discussion quite old https://github.com/itswadesh/svelte-commerce/discussions/26

How difficult to translate it. Is there some translation system already implemented so i can just add i18n dictionaries or English is hard code?

Can somebody explain the situation an point to right places in the code or structure?

**Comment by itswadesh** [comment]
Hi,

Thanks for showing interest.

Translations not implemented in this project yet. For now english is hard coded. i18n can be added, We can schedule a 30min call to help you understand the codebase, so that you can push translations.

**Reply by bouillon** [reply]
Thanks for quick response. I will come back to you after i played with it and managed the installation at all.
